RELEASES.md

```diff
@@ -1,3 +1,113 @@
+Version 1.24.0 (2018-02-15)
+==========================
+
+Language
+--------
+- [External `sysv64` ffi is now available.][46528]
+  eg. `extern "sysv64" fn foo () {}`
+
+Compiler
+--------
+- [rustc now uses 16 codegen units by default for release builds.][46910]
```

Comment:
I think it would be worth explicitly mentioning that this can result in worse performing binaries (#47745) and that you need to start using `codegen-units=1` for best performance.